p2p/pex/pex_reactor.go

```diff
@@ -189,8 +191,11 @@ func (r *PEXReactor) Receive(chID byte, src Peer, msgBytes []byte) {
 
 		// Seeds disconnect after sending a batch of addrs
 		if r.config.SeedMode {
-			// TODO: should we be more selective ?
-			r.SendAddrs(src, r.book.GetSelection())
+			// if node connects to seed, it does not have any trusted peers.
+			// Especially in the beginning, node should have more trusted peers than
+			// untrusted.
+			const biasToSelectNewPeers = 30 // 70 to select good peers
```

Comment:
Can this be moved to the top with the rest of the constants?

Reply:
what for?

Reply:
good to have all the constants in one place

Reply:
ok, will do. But I am gonna write a blog post and try to convince you that it's a good place if a const is not public and applied in one place.